client/src/styles/_fonts.scss

```diff
@@ -58,8 +58,10 @@
   content: "\6f";
 }
 
-.font-icon-plus:before {
-  content: "\6a";
+.font-icon-plus-thin:before {
+  content: "\44";
+  transform: rotate(45deg);
+  display: inline-block;
```

Comment:
Because this is one of the automated files for how we [update fonts](https://github.com/silverstripe/silverstripe-admin/pull/633) to admin it's probably best to add these styles to the button directly.